data/snippets.ts

```diff
@@ -920,7 +920,7 @@ export const snippetGroups = [
     icon: CalendarIcon,
   },
   {
-    name: "Misceallenous",
+    name: "Miscellaneous",
```

Comment:
Believe it or not, that was co pilot •‿• thanks! Maybe I should create a snippet for this haha